src/Admin/AdminExtensionInterface.php

```diff
@@ -165,6 +166,13 @@ public function alterObject(AdminInterface $admin, $object);
      */
     public function getPersistentParameters(AdminInterface $admin);
 
+    /**
+     * Get a chance to configure admin before used.
+     *
+     * @phpstan-param AdminInterface<T> $admin
+     */
```

Comment:
we could also add `// NEXT_MAJOR: Uncomment this method`

Reply:
I will uncomment this during the merge